drivers/counter/counter_gecko_rtcc.c

```diff
@@ -91,10 +91,12 @@ static u32_t counter_gecko_read(struct device *dev)
 }
 
 static int counter_gecko_set_top_value(struct device *dev, u32_t ticks,
```

Comment:
`ticks` are now a field of the `counter_top_cfg` structure. Does this compile?

Reply:
there is no test that uses that apparently. Fixed.